### DISCUSSION TITLE
[FEATURE] option . to subscribe locally, - emulated subscription

### DISCUSSION THREAD

**Discussion by DI555** [libre-tube/LibreTube#468] [General]
**Describe the feature you'd like to see added**
great would be to have possibility to subscribe locally in the app’s base, if you have not a yt account !
some kind of "emulation subscription" !!!

**Comment by FireMasterK** [comment]
The account system does not require a YT account, but a piped account.

**Comment by DI555** [comment]
this feature exactly for the case when no any accounts exist !
so, the local subscriptions whould be the great and much usable !

**Comment by FireMasterK** [comment]
When designing Piped, requiring an account was a design choice to improve efficiency, since the server could just keep track of subscription via pubsub.

In case of a client, constant polling is required for **each** subscription channel (like on NewPipe), which can take seconds or even minutes when combined altogether, and is unnecessary stress on instances.

I don't think this is a good idea to implement this.

**Comment by DI555** [comment]
ok, but its just convinience to using "local" subscription.
i think this issue may be paused so far..  until it’ll be obviouse for all that it’s quite comfortable..

but, i think i’ve got another way thought for that kind idea decision..

**Comment by Silther** [comment]
Can this discussion be closed since the feature is implemented?
